ethsigner/signer/multikey/src/test/java/tech/pegasys/ethsigner/signer/multikey/SigningMetadataTomlConfigLoaderTest.java

```diff
@@ -280,6 +280,28 @@ void hashicorpConfigIsLoadedIfHashicorpMetadataFileInDirectory() {
     assertThat(metadataFile.getConfig().getTimeout()).isEqualTo(50);
   }
 
+  @Test
+  void hashicorpConfigIsLoadedWithTlsDisabled() {
+    copyFileIntoConfigDirectory("hashicorpconfig_no_tls.toml");
+
+    final Collection<SigningMetadataFile> metadataFiles =
+        loader.loadAvailableSigningMetadataTomlConfigs();
+
+    assertThat(metadataFiles.size()).isOne();
+    assertThat(metadataFiles.toArray()[0]).isInstanceOf(HashicorpSigningMetadataFile.class);
+    final HashicorpSigningMetadataFile metadataFile =
+        (HashicorpSigningMetadataFile) metadataFiles.toArray()[0];
+
+    assertThat(metadataFile.getConfig().getSigningKeyPath()).isEqualTo("/path/to/key");
```

Comment:
nit: might be easier to read if metedataFie.getConfig() is refactored...
`subject = metadata.getConfig();
assertThat(subject.getSigningKeyPath()).isEqualTo(....`

Reply:
@rolfyone It is actually an almost ditto-copy of an existing test so I didn't even thought of making it more readable ;) .. but I will refactor it.